source/material_model/rheology/composite_visco_plastic.cc

```diff
@@ -123,6 +125,8 @@ namespace aspect
                                                                  const unsigned int composition,
                                                                  const SymmetricTensor<2,dim> &strain_rate,
                                                                  std::vector<double> &partial_strain_rates,
+                                                                 const double current_cohesion,
```

Comment:
Now that the drucker_prager_parameters are included in composite_visco_plastic, there is no need to pass around current_cohesion and current_friction_angle.

Reply:
Here the 2 lines can be removed.